fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentServiceClientTemplate.java

```diff
@@ -0,0 +1,138 @@
+                    .comment(comment -> {
+                        comment.description("Merges default client context with provided context.");
+                        comment.param("context", "the context to be merged with default client context.");
+                        comment.methodReturns("the merged context.");
+                    })
+                    .method(method -> {
+                        method.block("for (Map.Entry<Object, Object> entry : this.getContext().getValues().entrySet())", block -> {
+                            block.line("context = context.addData(entry.getKey(), entry.getValue());");
+                        });
+                        method.methodReturn("context");
+                    })
+                    .build();
+
+            MethodTemplate getLroResultMethod = MethodTemplate.builder()
+                    .imports(Arrays.asList(PollerFlux.class.getName(), PollResult.class.getName(),
+                            Mono.class.getName(), Flux.class.getName(), Response.class.getName(),
+                            ByteBuffer.class.getName(), Type.class.getName(),
+                            PollerFactory.class.getName()))
+                    .methodSignature("<T, U> PollerFlux<PollResult<T>, U> getLroResultAsync(Mono<Response<Flux<ByteBuffer>>> activationResponse, HttpPipeline httpPipeline, Type pollResultType, Type finalResultType)")
+                    .comment(comment -> {
+                        comment.description("Gets long running operation result.");
+                        comment.param("activationResponse", "the response of activation operation.");
+                        comment.param("httpPipeline", "the http pipeline.");
+                        comment.param("pollResultType", "type of poll result.");
+                        comment.param("finalResultType", "type of final result.");
+                        comment.param("<T>", "type of poll result.");
+                        comment.param("<U>", "type of final result.");
+                        comment.methodReturns("poller flux for poll result and final result.");
+                    })
+                    .method(method -> {
+                        method.methodReturn("PollerFactory.create(serializerAdapter, httpPipeline, pollResultType, finalResultType, defaultPollInterval, activationResponse)");
+                    })
+                    .build();
+
+            MethodTemplate getLroFinalResultOrErrorMethod = MethodTemplate.builder()
+                    .imports(Arrays.asList(PollerFlux.class.getName(), PollResult.class.getName(),
+                            Mono.class.getName(), AsyncPollResponse.class.getName(),
+                            ManagementError.class.getName(), ManagementException.class.getName(),
+                            LongRunningOperationStatus.class.getName(), SerializerEncoding.class.getName(), IOException.class.getName()))
```

Comment:
Desirable to automatically infer all (or most) of these import classes from code itself.